CHANGES.rst

```diff
@@ -1,6 +1,9 @@
 Changes in 0.8.1
 ----------------
 - Added Django 1.10 support `198 <https://github.com/django-money/django-money/issues/198>`_ (`Dmitry Dygalo <https://github.com/Stranger6667>`_)
+- Changed auto conversion of currencies using djmoney_rates (added in 0.7.3) to
```

Comment:
*doh* I already did the version bump in #201 

Sorry if you're getting conflicts now........